worker/reindex_reporting.json

```diff
@@ -1,21 +1,22 @@
 {
     "name": "reindex_reporting",
     "description": "Trigger a reindex of service data in mender-reporting.",
-    "version": 1,
+    "ephemeral": true,
+    "version": 3,
     "tasks": [
         {
             "name": "reindex_reporting",
-            "type": "http",
+            "type": "nats",
             "retries": 3,
```

Comment:
This parameter is unused.

Reply:
@alfrunes sorry, which parameter is unused?

Reply:
I think `retries` is unused, but I guess it has to be > 0 ?

Reply:
Sorry, I got confused. I thought since the error was suppressed/not returned in `processNATS` that it wouldn't retry.